packages/nextjs/components/builders/BuildersGrid.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { normalize } from "viem/ens";
+import { useEnsAvatar, useEnsName } from "wagmi";
+import { useBuildersEvent } from "~~/hooks/useBuildersEvent";
+import { BUILDER_DATA } from "~~/utils/constants";
+
+export function BuildersGrid() {
+  const { events: uniqueEvents, isLoading, error: errorReadingEvents } = useBuildersEvent();
+
+  const getBuilderData = (address: string) => {
+    return BUILDER_DATA[address as keyof typeof BUILDER_DATA];
+  };
+
+  // Component to handle ENS resolution for a single builder
+  const BuilderCard = ({ builderAddress }: { builderAddress: string }) => {
+    const { data: ensName } = useEnsName({
+      address: builderAddress as `0x${string}`,
+      chainId: 1,
+    });
+
+    const { data: ensAvatar } = useEnsAvatar({
+      name: ensName ? normalize(ensName) : undefined,
+      chainId: 1,
+    });
+
+    const builderData = getBuilderData(builderAddress);
+
+    const displayName = ensName || builderData.name;
+
+    return (
+      <div className="card w-86 h-122 shadow-lg hover:shadow-xl transition-shadow duration-300 rounded-xl overflow-hidden relative">
+        <Image
+          src={ensAvatar || "https://avatars.githubusercontent.com/u/111103306?v=4"}
```

Comment:
I don't think using a user-specific image as a general fallback image is a good idea.